examples/search-index/searchIndex.json

```diff
@@ -75,6 +75,14 @@
       "Value": {
         "Ref": "SearchIndex"
       }
+    },
+    "SearchIndexStatus": {
+      "Value": {
+        "Fn::GetAtt": [
+          "SearchIndex",
+          "Status"
+        ]
+      }
```

Comment:
@maastha FYI: to test that the changes work as expected, we need to use `Fn::GetAtt` to execute a read of the created resource